packages/label/README.md

```diff
@@ -0,0 +1,25 @@
+# label
+
+[![NPM version](https://img.shields.io/npm/v/label.svg)](https://www.npmjs.com/package/label)
+
+## Install
+
+[Yarn](https://yarnpkg.com/package/label):
+
+```sh
+yarn add label
+```
+
+[npm](https://www.npmjs.com/package/label):
+
+```sh
+npm install label --save
+```
+
+[UNPKG](https://unpkg.com/browse/label/)
+
+## Usage
+
+```ts
+import main from 'label'
```

Comment:
Can you add an example of how the component is used? See example in another README for inspiration

Reply:
Added an example